tools/src/test/resources/1.7/valid-license-expression-1.7.json

```diff
@@ -33,9 +33,21 @@
       ],
       "licenses": [
         {
-          "expression": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
+          "bom-ref": "my-license",
           "acknowledgement": "declared",
-          "bom-ref": "my-license"
+          "expression": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
+          "expression-texts": [
+            "license-identifier": "EPL-2.0",
```

Comment:
went with the name `license-identifier` - as proposed via https://github.com/CycloneDX/specification/issues/554#issuecomment-2552934734

the name is disputable and not final.
please comemnt below, whichname you find more fitting.